spec/models/category_spec.rb

```diff
@@ -4,8 +4,38 @@
   it { should have_many(:videos) }
 
   describe '#recent_videos' do
-    it 'should return an array of all videos if the total number is less than six'
-    it 'should return an array of six videos if the total number of videos is at least six'
-    it 'returns the video array in reverse chronological order'
+    it 'should return an array of all videos if the total number is less than six' do
```

Comment:
You should not use the word `should` in examples message. Read more [here](http://www.betterspecs.org/#should).